orcestra_book/scienceseminars.md

```diff
@@ -14,11 +14,13 @@ On ground days we offer science talks to share the work of the scientists.
 | 19 Aug, 11 LT   |   Marius Rixen (MPI-M)           | What controls  humidity haloes surrounding trade wind clouds?     |
 |                 |   Clara Bayley (MPI-M)           | Modelling rain evaporation during EUREC$^4$A       |
 |                 |                |                 |
-| 23 Aug, 11 LT   |   Martin Singh  (Monash University)    | ITCZ edge intensification: insight from simplified modelling       |
-|                 |   Basile Poujol (LMD)     |   *tbd*      |
+| 23 Aug, 10.15 LT|   Martin Singh  (Monash University)    | ITCZ edge intensification: insight from simplified modelling       |
+|                 |   Basile Poujol (LMD)     |    Convective clouds as dipoles    |
 |                 |                |                 |
-| 26 Aug, 11 LT   |   Paula Romero Jure (University of Leeds)      | Physical drivers of high clouds' radiative effects in the Tropical Pacific   |
+| 26 Aug, 10.15 LT|   Paula Romero Jure (University of Leeds)      | Physical drivers of high clouds' radiative effects in the Tropical Pacific   |
```

Comment:
The calendar needs to be adjusted as well 😉

Reply:
I'm on it. Thanks! 😄